src/xma/include/lib/xmahw.h

```diff
@@ -47,12 +49,23 @@ typedef struct XmaHwKernel
     uint32_t    ddr_bank;
     //For execbo:
     int32_t     kernel_complete_count;
+    std::unique_ptr<std::condition_variable> kernel_complete_count_cv;
```

Comment:
Why pointer?  Also, I don't see the condition variable used any where?

Reply:
I will remove unused condition_variable.
I'm changing c code to c++ in small portions. Later I may change arrays to vector so I want to keep it as pointer.

Reply:
changed